test/cpp/end2end/xds_end2end_test.cc

```diff
@@ -764,13 +764,14 @@ class XdsEnd2endTest : public ::testing::TestWithParam<TestType> {
   }
 
   void WaitForBackend(size_t backend_idx, bool reset_counters = true) {
-    gpr_log(GPR_INFO,
-            "========= WAITING FOR BACKEND %lu ==========", backend_idx);
+    gpr_log(GPR_INFO, "========= WAITING FOR BACKEND %lu ==========",
+            static_cast<unsigned long>(backend_idx));
```

Comment:
For future reference, a better approach here is to change the format string to use `PRIuPTR` instead of `%lu`.